docker-compose.yml

```diff
@@ -11,23 +11,29 @@ services:
     environment:
       - HOST=0.0.0.0
       - PORT=8002
-      - APP_URL=https://localhost:8002
+      - APP_URL=http://localhost:8002
       - APP_SETTINGS=/app/settings.py
       - ENV=localdev
-      - DATABASE_URL=postgres://tooltooluser:tooltoolpassword@db:80020/tooltooldb
+      - DATABASE_URL=postgres://tooltooluser:tooltoolpassword@db/tooltooldb
+      # "notasecret"
+      - SECRET_KEY=bm90YXNlY3JldA==
       - TASKCLUSTER_SECRET=$TASKCLUSTER_SECRET
       - TASKCLUSTER_CLIENT_ID=$TASKCLUSTER_CLIENT_ID
       - TASKCLUSTER_ACCESS_TOKEN=$TASKCLUSTER_ACCESS_TOKEN
+      # S3_REGIONS format is "foo:bar; blah:blah", which turns into a dict in the app
+      - S3_REGIONS=$S3_REGIONS
```

Comment:
If `S3_REGIONS` doesn't contain any secrets, maybe put some default "localdev" value, so it'd be easier to find the corresponding credentials?

Reply:
Maybe. That value is going to be dependent on whatever buckets you set up with your own access keys though, I think? I guess we could set a default and add some instructions about how to set up buckets, credentials, etc.